.github/workflows/ci.yaml

```diff
@@ -1,115 +1,118 @@
-      #  run: |
-      #    echo "::group::Fixup Composer platform config for third-parties deps not PHP 8 ready yet"
-      #      composer config platform.php 7.4.99
-      #    echo "::endgroup::"
-
-      - name: 'Install dependencies'
-        run: |
-          echo "::group::composer update"
-            composer update --no-progress --ansi
-          echo "::endgroup::"
-
-          echo "::group::install phpunit"
-            vendor/bin/simple-phpunit install
-          echo "::endgroup::"
-
-      - name: 'Run tests'
-        run: vendor/bin/simple-phpunit --testdox
-
+    lint:
+        name: Lint
+        runs-on: 'ubuntu-latest'
+        timeout-minutes: 5
+
+        steps:
+            - name: 'Checkout'
+              uses: actions/checkout@v2
+
+            - name: 'Setup PHP'
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '8.3'
+
+            - name: 'Install dependencies'
+              run: |
+                make install
+                vendor/bin/simple-phpunit --version
+
+            - name: 'Check style'
+              run: make lint
+
+    test:
+        name: ${{ matrix.name }}
+        runs-on: ${{ matrix.os }}
+        timeout-minutes: 8
+        continue-on-error: ${{ matrix.allow-failure == 1 }}
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    # Lowest deps
+                    - name: 'Test lowest deps Symfony 5.4 [Linux, PHP 7.4]'
+                      os: 'ubuntu-latest'
+                      php: '7.4'
+                      symfony: '5.4.*@dev'
+                      composer-flags: '--prefer-lowest'
+                      allow-unstable: true
+
+                    # Most recent versions
+                    - name: 'Test Symfony 5.4 [Linux, PHP 8.0]'
+                      os: 'ubuntu-latest'
+                      php: '8.0'
+                      symfony: '5.4.*@dev'
+                      allow-unstable: true
+
+                    - name: 'Test Symfony 6.0 [Linux, PHP 8.1]'
+                      os: 'ubuntu-latest'
+                      php: '8.1'
+                      symfony: '6.0.*@dev'
+                      allow-unstable: true
+
+                    - name: 'Test next Symfony 6.4 [Linux, PHP 8.2]'
+                      os: 'ubuntu-latest'
+                      php: '8.2'
+                      symfony: '6.4.*@dev'
+                      allow-unstable: true
+
+                    # Bleeding edge (unreleased dev versions where failures are allowed)
+                    - name: 'Test next Symfony [Linux, PHP 8.3] (allowed failure)'
+                      os: 'ubuntu-latest'
+                      php: '8.3'
+                      symfony: '7.0.*@dev'
+                      composer-flags: '--ignore-platform-req php'
+                      allow-unstable: true
+                      allow-failure: true
+
+        steps:
+            - name: 'Set git to use LF'
```

Comment:
❔ **question:** For my information, why is it necessary in this repository?

Reply:
It's not, just bad copy/paste 🫣